include/test/mir/test/doubles/mock_cursor.h

```diff
@@ -36,7 +36,7 @@ struct MockCursor : public mir::graphics::Cursor
     MOCK_METHOD1(move_to, void(mir::geometry::Point));
     MOCK_METHOD1(scale, void(float));
     MOCK_METHOD0(renderable, std::shared_ptr<mir::graphics::Renderable>());
-    MOCK_METHOD0(needs_compositing, bool());
+    MOCK_CONST_METHOD0(needs_compositing, bool());
```

Comment:
We're trying to move away from the ...[digit] version of this macro

Reply:
Come on! You didn't need to change this, but as you chose to, you could have fixed the surrounding code